packages/shared/src/components/post/PostContent.tsx

```diff
@@ -0,0 +1,386 @@
+  navigation?: Omit<PostNavigationProps, 'postActionsProps'>;
+  post?: Post;
+  position?: CSSProperties['position'];
+  isLoading?: boolean;
+  onScroll?: UIEventHandler<HTMLDivElement>;
+}
+
+const BodyContainer = classed(
+  'div',
+  'flex flex-col tablet:flex-row max-w-[100vw] overflow-y-auto h-full pb-6',
+);
+
+const PageBodyContainer = classed(
+  BodyContainer,
+  'm-auto w-full max-w-[63.75rem]',
+);
+
+const PostContainer = classed(
+  'main',
+  'flex flex-col flex-1 px-8 tablet:pb-20 laptop:max-w-[40rem] tablet:border-r tablet:border-theme-divider-tertiary',
+);
+
+export function PostContent({
+  id,
+  seo,
+  postData,
+  className,
+  isFallback,
+  enableAuthorOnboarding,
+  enableShowShareNewComment,
+  navigation,
+  isFetchingNextPage,
+  isModal,
+  onClose,
+}: PostContentProps): ReactElement {
+  if (!id && !isFallback) {
+    return <Custom404 />;
+  }
+
+  const { user, showLogin, tokenRefreshed } = useContext(AuthContext);
+  const { trackEvent } = useContext(AnalyticsContext);
+  const [parentComment, setParentComment] = useState<ParentComment>(null);
+  const [showShareNewComment, setShowShareNewComment] = useState(false);
+  const [lastScroll, setLastScroll] = useState(0);
+  const [authorOnboarding, setAuthorOnboarding] = useState(false);
+  const [upvotedPopup, setUpvotedPopup] = useState(getUpvotedPopupInitialState);
+  const queryClient = useQueryClient();
+  const postQueryKey = ['post', id];
+  const [position, setPosition] =
+    useState<CSSProperties['position']>('relative');
+  const {
+    data: postById,
+    isLoading,
+    isFetched,
+  } = useQuery<PostData>(
+    postQueryKey,
+    () => request(`${apiUrl}/graphql`, POST_BY_ID_QUERY, { id }),
+    { initialData: postData, enabled: !!id && tokenRefreshed },
+  );
+
+  useEffect(() => {
+    if (enableAuthorOnboarding) {
+      setAuthorOnboarding(true);
+    }
+  }, [enableAuthorOnboarding]);
+
+  useEffect(() => {
+    if (enableShowShareNewComment) {
+      setTimeout(() => setShowShareNewComment(true), 700);
+    }
+  }, [enableShowShareNewComment]);
+
+  const handleShowUpvotedPost = (upvotes: number) => {
+    setUpvotedPopup({
+      modal: true,
+      upvotes,
+      requestQuery: {
+        queryKey: ['postUpvotes', id],
+        query: POST_UPVOTES_BY_ID_QUERY,
+        params: { id, first: DEFAULT_UPVOTES_PER_PAGE },
+      },
+    });
+  };
+
+  const handleShowUpvotedComment = (commentId: string, upvotes: number) => {
+    setUpvotedPopup({
+      modal: true,
+      upvotes,
+      requestQuery: {
+        queryKey: ['commentUpvotes', commentId],
+        query: COMMENT_UPVOTES_BY_ID_QUERY,
+        params: { id: commentId, first: DEFAULT_UPVOTES_PER_PAGE },
+      },
+    });
+  };
+
+  useSubscription(
+    () => ({
+      query: POSTS_ENGAGED_SUBSCRIPTION,
+      variables: {
+        ids: [id],
+      },
+    }),
+    {
+      next: (data: PostsEngaged) => {
+        queryClient.setQueryData<PostData>(postQueryKey, (oldPost) => ({
```

Comment:
Keep in mind to not overwrite the fix we did today.

Reply:
This should be fixed by the recent merge now. Thank you for pointing it out!